src/rmUnused.ml

```diff
@@ -36,8 +36,8 @@
 
  *)
 
-(* rmtmps.ml *)
```

Comment:
Maybe we can add some documentation comment to this module to indicate that this is a rename of `RmTemps`?